homeassistant/components/aranet/sensor.py

```diff
@@ -92,7 +92,7 @@ class AranetSensorEntityDescription(SensorEntityDescription):
         key="radiation_rate",
         translation_key="radiation_rate",
         name="Radiation Dose Rate",
-        native_unit_of_measurement="μSv/h",
+        native_unit_of_measurement="μSv/h",  # "μ" == "\u03bc"
```

Comment:
Why do we need the comment `# "μ" == "\u03bc"`, doesn't the pylint checker catch it?

Reply:
I'll remove the comments. The pylint checker checks the value.